packages/front-end/services/config.ts

```diff
@@ -83,6 +124,9 @@ export function useConfigJson({
         "userIdType",
         "userIdTypes",
         "tags",
+        "denominator",
+        "type",
+        "conditions",
```

Comment:
Add `queryFormat` to this list and get rid of the `if (m.sql) ... else ...` block below.  Instead, always include all of the columns.  So that means add `sql`, `anonymousIdColumn`, etc. to this fields array as well.